deck/templates/event/event_create_schedule.html

```diff
@@ -55,66 +55,77 @@
         <p class="text-center text-muted">{% trans "Approved talks" %}</p>
         <div id="event-approved-proposals" class="proposal-container">
           {% for activity in event.get_schedule %}
-            {% if activity.activity_type == 'proposal' %}
-              <div id="{{ activity.slug }}"  data-href="{% url 'api_event_activity' event_slug=event.slug slug=activity.slug %}" class="panel proposal-item panel-success">
-                <div class="panel-body">
-                  <input checked="checked" name="approved_activities" value="{{ activity.pk }}" class="hide" type="checkbox" />
-                  <div class="pull-right proposal-actions">
-                    <img src="{{ activity.author|get_user_photo }}" alt="author photo" class="img-circle author-photo hide">
-                    <button type="button" class="btn-flat gray text-upper update-activity">
-                      <i class="icon-pencil"></i>
-                    </button>
-                  </div>
+              {% if activity.proposal.is_approved %}
```

Comment:
For me, the condition `activity.proposal.is_approved` should be together if `activity.activity_type == 'proposal'` on the next line, something like: `activity.activity_type == 'proposal' and activity.proposal.is_approved`, otherwise it will give you an exception for trying to get the attribute `is_approved` from a activity that is not a proposal, it means that will only show the Proposals.

See the difference: 
![screen shot 2017-09-29 at 12 18 05](https://user-images.githubusercontent.com/1490875/31022795-4a4d3b66-a510-11e7-84b0-aec74c2650e9.png)

And with `activity.activity_type == 'proposal' and activity.proposal.is_approved`:

![screen shot 2017-09-29 at 12 18 12](https://user-images.githubusercontent.com/1490875/31022809-5b7492f4-a510-11e7-9e66-63459af2a7b4.png)

Reply:
```activity.activity_type == 'proposal' and activity.proposal.is_approved``` seems to be causing issues as it will go to the else part even thought it's a proposal.

![image](https://user-images.githubusercontent.com/9161520/31026557-960df00a-a564-11e7-892c-9e3f1f275cf7.png)

I guess a nested condition like this would work:
```{% if activity.activity_type == 'proposal' %}
                {% if activity.proposal.is_approved %}```